sys/dbgpin/dbgpin.c

```diff
@@ -0,0 +1,18 @@
+
+
+#include "dbgpin.h"
+
+const gpio_t dbgpin_pins[] = { DBGPIN_PINS };
+
+unsigned dbgpin_cnt(void)
+{
+    return ARRAY_SIZE(dbgpin_pins);
+}
```

Comment:
This should be a compile time constant, so IMO this should either be a `static inline` function in the header (preferably) or just a macro. You probably want to use `size_t` as type.

Reply:
does not work -> `ARRAY_SIZE` does not work on `extern const dbgpin_pins[];` in the header file.

Reply:
```C
static inline size_t dbgpin_count(void)
{
    static const gpio_t will_be_optimized_out[] = { DBGPIN_PINS };
    return ARRAY_SIZE(will_be_optimized_out);
}